spec/index.html

```diff
@@ -98,7 +98,7 @@
   Turtle [[RDF12-TURTLE]] format.
   </p>
 
-  <p>RDF 1.2 TriG shares <a>quoted triples</a> with [[RDF12-TURTLE]]
+  <p>RDF 1.2 TriG shares <a data-cite="RDF12-CONCEPTS#dfn-triple-term">triple terms</a> with [[RDF12-TURTLE]]
     as a fourth kind of <a data-cite="RDF12-CONCEPTS#dfn-rdf-term">RDF term</a>
     which can be used as the
     <a data-cite="RDF12-CONCEPTS#dfn-subject">subject</a> or
```

Comment:
This allows triple term as subject.

Reply:
Something was lost in a rebase, I think.